age/keysource.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const (
+	privateKeySizeLimit = 1 << 24 // 16 MiB
```

Comment:
Is this still needed? It looks like a left-over to me.

Reply:
A reference to the [age source](https://github.com/FiloSottile/age/blob/101cc8676386b0503571a929a88618cae2f0b1cd/parse.go#L24) that was never used, as best I can determine. I don't see a role for key size limits in serialization routines...